hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -1946,6 +1946,14 @@ possible configurations would overwhelm and obscure the important.
       responses with complete data.
     </description>
   </property>
+  <property>
+    <name>hbase.master.metafixer.max.merge.count</name>
+    <value>10</value>
```

Comment:
The default changed.

Reply:
@saintstack I have changed the  default value

Thanks